geniza/templates/snippets/language_switcher.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+
+{% if LANGUAGES|length > 1 %}
+    <form id="language" action="{% url 'set_language' %}" method="post">{% csrf_token %}
+        <fieldset>
```

Comment:
I feel like we should add a label here; although I know this implementation is provisional. What do you think about "read this page in" [language] for now?